src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,18 @@
+import './TodoInfo.scss';
+
+import { UserInfo } from '../UserInfo/UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+export const TodoInfo = ({ todo }) => {
+  const className = todo.completed === true
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo';
```

Comment:
use classnames library for this or use more meaningful variable naming

Reply:
+